frontend/src/utils/DropDownAutoCompleteTextArea.svelte

```diff
@@ -0,0 +1,82 @@
+<style>
+  .dropdown {
+    text-align: left;
+    position: absolute;
+    margin-top: 3px;
+  }
+
+  textarea {
+    background: inherit;
+    color: inherit;
+    border: none;
+    border-bottom: 1px solid white;
```

Comment:
Since this is many lines long, I wonder if it makes sense to style with a fully surrounding border instead of just an underline? Don't feel strongly either way, but thought I would throw it out there